src/theme/theme.ts

```diff
@@ -94,7 +94,7 @@ const newColors: { [key in MY_OBJECT_KEYS]: Color } = {
 	simplyGreen: {}
 };
 
-const opacities = [1, 0.8, 0.6, 0.4, 0.2];
+const opacities = [1, 0.8, 0.6, 0.4, 0.2, 0.1];

```

Comment:
Let's keep it on multiples of 2. If design needs an opacity that's not here they need to update the figma first.